src/Common/tests/Scripts/Tools/net_providers.txt

```diff
@@ -0,0 +1,12 @@
+"{bdd9a83e-1929-5482-0d73-2fe5e1c0e16d}"
```

Comment:
Is there any way to document in this file which GUID maps to which System.Net library? Or have a different file to describe a mapping of GUID <--> text?